### PR TITLE
Added Section Descriptions Part 2

### PR DESCRIPTION
**Pull request by KieranRobson**
Part 2 to #3180 
Towards #2984 

Descriptions were modified and taken from Wikipedia with all hyperlinks leading to respective wikipedia pages

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1621,6 +1645,8 @@ _Related: [Booking and Scheduling](#booking-and-scheduling)_
 
 **[`^        back to top        ^`](#)**
 
+Software and tools for managing [recipes](https://en.wikipedia.org/wiki/Recipe)
```
Comment (nodiscc) [inline README.md:1648]: Missing `.` at the end of the description

File: README.md [hunk 2]
```diff
@@ -1653,6 +1683,8 @@ _Related: [Booking and Scheduling](#booking-and-scheduling)_
 
 **[`^        back to top        ^`](#)**
 
+A [search engine](https://en.wikipedia.org/wiki/Search_engine) is a software system designed to carry out [web searches](https://en.wikipedia.org/wiki/Web_query).
```
Comment (nodiscc) [inline README.md:1686]: I think the correct description would be that one (these are not only Web search engines):

```
A [search engine](https://en.wikipedia.org/wiki/Search_engine_(computing)) is an [information retrieval system](https://en.wikipedia.org/wiki/Information_retrieval) designed to help find information stored on a computer system. This includes [Web search engines](https://en.wikipedia.org/wiki/Web_search_engine).
```

File: README.md [hunk 3]
```diff
@@ -1773,6 +1807,8 @@ _Related: [Booking and Scheduling](#booking-and-scheduling)_
 
 **[`^        back to top        ^`](#)**
 
+[Localization](https://en.wikipedia.org/wiki/Internationalization_and_localization) is the proccess of adapting code and software to other langauages.
```
Comment (nodiscc) [inline README.md:1810]: typo `langauages`

File: README.md [hunk 4]
```diff
@@ -1885,6 +1931,8 @@ _Related: [Software Development - Project Management](#software-development---pr
 
 **[`^        back to top        ^`](#)**
 
+[Help desk](https://en.wikipedia.org/wiki/Help_desk_software) and [issue tracking](https://en.wikipedia.org/wiki/Issue_tracking_system) software to keep track of user requests and deal with other customer-care-related issues.
```
Comment (nodiscc) [inline README.md:1934]: As this section mixes support/helpdesk-oriented ticketing systems, and bug/issue trackers, I think the description could be improved (track user requests, bugs, missing features...). Right now I can't think of a correct way to phrase it.

File: README.md [hunk 5]
```diff
@@ -1919,6 +1969,8 @@ _Related: [Task Management & To-do Lists](#task-management--to-do-lists), [Softw
 
 **[`^        back to top        ^`](#)**
 
+[URL shortening](https://en.wikipedia.org/wiki/URL_shortening) is the action of shortening a [URL](https://en.wikipedia.org/wiki/Uniform_Resource_Locator) may be made substantially shorter and still direct to the required page.
```
Comment (nodiscc) [inline README.md:1972]: `may be made` -> `to make it` ?

File: README.md [hunk 6]
```diff
@@ -1956,6 +2008,8 @@ Before hosting one, please see [shortcomings](https://en.wikipedia.org/wiki/URL_
 
 **[`^        back to top        ^`](#)**
 
+Tools and software to host a personal [Wiki](https://en.wikipedia.org/wiki/Wiki).
```
Comment (nodiscc) [inline README.md:2011]: Let's just copy the description from wikipedia?

```
A [wiki](https://en.wikipedia.org/wiki/Wiki) is a publication collaboratively edited and managed by its own audience directly using a web browser.
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (KieranRobson) [reply README.md:1934]: `[Ticketing](https://en.wikipedia.org/wiki/Issue_tracking_system) and [Bug Tracking](https://en.wikipedia.org/wiki/Bug_tracking_system) software to help the tracking of user requests, bugs and missing features`

What about the above?

Comment (nodiscc) [reply README.md:1911]:
```
[Helpdesk](https://en.wikipedia.org/wiki/Help_desk_software), [bug](https://en.wikipedia.org/wiki/Bug_tracking_system) and [issue](https://en.wikipedia.org/wiki/Issue_tracking_system) tracking software to help the tracking of user requests, bugs and missing features.
```

^ ?

Comment (KieranRobson) [reply README.md:1911]: Yeah thats more fluent than what I had 😆, will add now and resolve

**Review by nodiscc — APPROVED** [review]
Thanks!

**Pull request merged**